packages/edit-site/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -74,7 +74,7 @@ export default function ListViewSidebar() {
 				<BlockNavigationTree
 					blocks={ rootBlocks }
 					selectBlock={ selectEditorBlock }
-					selectedBlockClientId={ selectedBlockClientId }
+					selectedBlockClientId={ selectedBlockClientIds }
```

Comment:
Would we maybe want to update the param here and elsewhere to reflect its new, consistently plural state? Eg `selectedBlockClientIds={ selectedBlockClientIds }` ?

Reply:
Thanks for raising this, I was undecided if pursuing it, but in the end it wasn't that complicated.

In 51631bd I've replaced the singular prop with the plural.
Since the `BlockNavigationTree` component is exposed by the package, I added a `deprecate` message and a little prop conversion for backward compatibility.

(Note: the main `BlockNavigation` component is **not** exported, reason why I handled the deprecation in the sub-component `BlockNavigationTree`.)

I've also updated all the consumers of these components to the new array, plural prop.

As the multi-selection is currently only supposed to be supported by the persistent List View, I've chosen to keep the pre-existing behaviour for the other consumers (dropdown List View, and Navigation block's Block Navigation).
Namely: they only highlight *singularly selected blocks*, while multi-selected blocks are simply ignored.
The technical reason for this is that single and multiple selections refer to different state properties.